tests/morpheus/stages/test_databricks_deltalake_source_stage.py

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+try:
+    import databricks.connect  # pylint: disable=unused-import # noqa
+except ImportError:
+    import pytest
+    pytest.skip("databricks-connect not installed", allow_module_level=True)
```

Comment:
Ideally this would use a fixture similar to the `nemollm_fixture` in `tests/conftest.py` which would allow us to optionally fail if the `--fail_missing` is used. Otherwise we could end up with a test that is never run.

Reply:
Ack